src/smt/proof_post_processor_dsl.h

```diff
@@ -62,6 +62,8 @@ class ProofPostprocessDsl : protected EnvObj, public ProofNodeUpdaterCallback
   Node d_true;
   /** The rewrite database proof generator */
   rewriter::RewriteDbProofCons d_rdbPc;
+  /** The accumulated subgoals from calls to d_rdbPc */
+  std::vector<std::shared_ptr<ProofNode>> d_subgoals;
```

Comment:
I can get behind this one because it's part of a top-level utility. But I would still prefer to force ourselves to address subgoals at each step they're introduced (I know I am being somewhat paranoid here with all the tests).